apps/civicsignalblog/src/lib/data/json/applications.js

```diff
@@ -0,0 +1,12 @@
+const applications = [
+  "CivicSignal",
+  "Explorer",
+  "Topic Mapper",
+  "Source Manager",
+  "Research Blog",
+].map((label) => ({
+  label,
+  value: label.toLowerCase().replace(/\s+/g, "_"),
+}));
+
+export default applications;
```

Comment:
Not sure why this is in `lib/data/json`... it's certainly not a JSON.

Reply:
Has this been addressed @m453h ?

Reply:
@kilemensi, I think much like `lib/data/json/countries.js` (which is currently not being referenced anywhere), I propose to move them to the `lib/data/common`

Reply:
Decided to move it to payload/lib/data/common